complaint_search/tests/expected_results/search_with_size_corrected__valid.json

```diff
@@ -21,7 +21,7 @@
     "timely",
     "zip_code"
   ],
-    "size": 500,
+    "size": 100,
```

Comment:
Was the purpose of this test to see an "invalid" number and then correct on the API side, or was it supposed to contain a valid size?

Reply:
Yeah, the call has 500 as the size and the request is corrected to 100

Reply:
```python
    def test_search_with_size_corrected__valid(self):
        self.request_test("search_with_size_corrected__valid", size=500)
```